### PR TITLE
Added Photoswipe and jQuery-UI CSS minification

### LINKED ISSUE #24959: Photoswipe.css and photoswipe-default-skin.css comes unminified
[Photoswipe.css](https://github.com/woocommerce/woocommerce/blob/master/assets/css/photoswipe/photoswipe.css) and [photoswipe-default-skin.css](https://github.com/woocommerce/woocommerce/blob/master/assets/css/photoswipe/default-skin/default-skin.css) enqueued on product page comes unminified. Files compressed over https://csscompressor.com/ are half a size (15kb saved).

### PR DESCRIPTION
**Pull request by ObliviousHarmony**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

I added the CSS files for Photoswipe and jQuery-UI to the Grunt minification task, and updated the enqueue for the styles.

Closes #24959.

### How to test the changes in this Pull Request:

1. Run an `npm build` to generate the resources.
2. Note the presence of minified files `assets/css/photoswipe/photoswipe.min.css`, `assets/css/photoswipe/default-skin/default-skin.min.css`, `assets/css/jquery-ui/jquery-ui.min.css`, and `assets/css/jquery-ui/jquery-ui-rtl.min.css`.
3. View the source of a product page and note the correct inclusion of `photoswipe.min.css`

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Fix: Added appropriate minification to photoswipe.css.

### CODE REVIEW

**Comment by kloon** [comment]
I am not sure we should be adding this? Since Photoswipe is a library we include we should not be minifying the files on our build process, but instead just loaded the minified files included with the library.

The same goes for any changes, we do not make changes in our repo for libraries but instead we will just pull in the latest versions from upstream and include it in our release.

If the files are missing then we should include those and make sure we load them when not using SCRIPT_DEBUG.

Open to suggestions though.

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,9 @@ none
 
 # Compiled CSS
 /assets/css/*.css
+/assets/css/photoswipe/*.min.css
+/assets/css/photoswipe/default-skin/*.min.css
+/assets/css/jquery-ui/*.min.css
```
Comment (kloon) [inline .gitignore]: Lets leave this out since the library already includes a minified file.

File: .gitignore [hunk 2]
```diff
@@ -20,6 +20,9 @@ none
 
 # Compiled CSS
 /assets/css/*.css
+/assets/css/photoswipe/*.min.css
```
Comment (kloon) [inline .gitignore]: Photoswipe can stay since they do not include minified files but sass files which we compile.

File: Gruntfile.js [hunk 3]
```diff
@@ -148,11 +148,36 @@ module.exports = function( grunt ) {
...
+				files: [
+					{
+						expand: true,
+						cwd: '<%= dirs.css %>/',
+						src: ['*.css'],
+						dest: '<%= dirs.css %>/',
+						ext: '.css'
+					},
+					{
+						expand: true,
```
Comment (kloon) [inline Gruntfile.js]: Let's not minify a library that already includes a minified file in the repo.

**Review by kloon — DISMISSED** [review]

**Comment by peterfabian** [comment]
> Since Photoswipe is a library we include we should not be minifying the files on our build process, but instead just loaded the minified files included with the library.
I'd just like to note that we do modify the Photoswipe CSS with our own code (first ±30 lines in the css file). 
There might be a better way to handle this (e.g. pull Photoswipe from source and have our CSS file separately?), but that's maybe for another day.

**Comment by ObliviousHarmony** [comment]
Hehe, sorry! I come from a mindset of compiling everything! I've gone ahead and removed the JQuery-UI minification. 

I agree that the best course of action would be to just store the minified asset and move our override styles out of the original file. I don't see a reason to hold off on this, so I think I'm going to just move ahead with that.

I've also noticed that our `jquery-ui.css` file seems to have been broken at some point in its lifespan. It looks like some build process caused it to contain the same styles 68 times. If there's no issue with it, I'll just replace the `jquery-ui.css` and `jquery-ui.min.css` with the correct files.

I do think a more appropriate solution to our custom photoswipe styling would be to move it into its own file. It also seems like our JQuery-UI file contains the same styles repeatedly. Perhaps this is some kind of build artifact? 

Would anyone have a problem with me replacing the

**Review by claudiosanches — APPROVED** [review]
Since we include some custom CSS in photoswipe I agree that we could just minify those files for now.
We may revisit those scripts in the future in case we remove jQuery.

**Pull request merged**